Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -25,10 +25,12 @@
 
 enum AspectMode
 {
-	ASPECT_AUTO       = 0,
-	ASPECT_FORCE_16_9 = 1,
-	ASPECT_FORCE_4_3  = 2,
-	ASPECT_STRETCH    = 3,
+	ASPECT_AUTO        = 0,
+	ASPECT_FORCE_16_9  = 1,
+	ASPECT_FORCE_4_3   = 2,
+	ASPECT_STRETCH     = 3,
+	ASPECT_ANALOG      = 4,
+	ASPECT_ANALOG_WIDE = 5,
```

Comment:
I think the "analog" options can replace the equivalent "force" options:

Auto = 0 (Force 1:1 Pixels)
Analog Wide = 1
Analog = 2
Stretch = 3

This will aid backwards compatibility of the settings.

To render a true 1:1 pixel display, shouldn't VideoInterface return num horizontal pixels and num vertical pixels to the renderer rather than a float aspect ratio?  This way, Auto would render the "blank" pixels and stretching to a 4:3 window would render the screen in letterbox format.

In case you have not read it already, see: http://code.google.com/p/dolphin-emu/issues/detail?id=7948#c3